### DISCUSSION TITLE
Starthilfe mit HM-CFG-LAN

### DISCUSSION THREAD

**Discussion by rhegner** [OpenCCU/OpenCCU#2476] [🇩🇪 Fragen&Antworten]
Hello everyone....

Ich bin ganz neu hier und brauche etwas Starthilfe.

Seit vielen Jahren habe ich FHEM mit einem HM-CFG-LAN (das runde Ding) erfolgreich am Laufen.

Nun möchte ich mein Setup etwas modernisieren und habe mich für Home Assistant mit dem RaspberryMatic Plugin entschieden. Da mir momentan noch die dedizierte Hardware fehlt, habe ich die Testumgebung wie folgt aufgebaut:

Windows11 --> HyperV --> Home Assistant --> RaspberryMatic Plugin

Die Installation hat soweit auch gut geklappt, und ich konnte das LAN-Interface in der Systemsteuerung eintragen:

![image](https://github.com/jens-maus/RaspberryMatic/assets/2563997/cebf89c0-a4db-4966-8be7-c85c49721ac2)

Die IP-Adresse ist für RaspberryMatic erreichbar, das habe ich mit "Skript testen" aus RaspberryMatic heraus ausprobiert

    string stderr;
    string stdout;
    system.Exec("ping -w 10 192.168.0.4",&stdout, &stderr);
    WriteLine (stdout);
    WriteLine (stderr);

Doch hier komme ich nicht weiter: es wird kein Status angezeigt, ich sehe keine Devices, ich weiss gar nicht, ob RaspberryMatic nun bereits versucht, mit dem LAN-Interface Verbindung aufzunehmen, oder wo ich Logs von allfälligen Verbindungsversuchen einsehen kann.

Die einzige Fehlermeldung, die ich gelegentlich sehe, ist folgende:

![image](https://github.com/jens-maus/RaspberryMatic/assets/2563997/daf3a838-92a9-40e4-8163-bd4320d4064f)

Viel weiter bringt mich das aber auch nicht.

Deshalb meine Frage: was sind jetzt die nächsten Schritte um sicherzustellen, dass die Verbindung zum LAN-Interface funktioniert, und um die gepairten Devices ins RaspberryMatic zu bringen?

**Comment by rhegner** [comment]
Wie [hier](https://github.com/jens-maus/RaspberryMatic/discussions/1162#discussioncomment-5900480) angeregt habe ich das LAN-Interface jetzt auch in FHEM explizit geschlossen. Leider sehe ich immer noch keinen Status in RaspberryMatic.

Nach dem Klick auf "Übernehmen" habe ich übrigens auch RaspberryMatic neu gestartet, wie verlangt.

**Comment by rhegner** [comment]
Ich hatte den falschen Interface-Typ ausgewählt. Jetzt funktionierts :)

**Reply by bangkokopete** [reply]
hi, which interface type did you finally use to have it running ? thanks

**Reply by rhegner** [reply]
This type works for me: HM Configuration Tool LAN

**Reply by bangkokopete** [reply]
thx, will try an report back

**Reply by bangkokopete** [reply]
Hallo wird im Statusbildschirm angezeigt.
Nicht vergessen in der RM firewall den port 1000 freigeben. dann gehts auch 
habe vorsichtshalber port 1001,2000 und 2001 auch freigeben 
danke vielmals
Peter
